.aicoe-ci.yaml

```
@@ -5,9 +5,8 @@ check:
   - thoth-build
 # Uncomment following lines to build a public image of this repo
 build:
-   build-stratergy: Source
-   build-source-script: "image:///opt/app-root/builder"
-   base-image: quay.io/thoth-station/s2i-elyra-custom-notebook:latest
+  build-stratergy: Dockerfile
+  dockerfile-path: Dockerfile
    custom-tag: latest
    registry: quay.io
    registry-org: os-climate
```

```suggestion
  custom-tag: latest
  registry: quay.io
  registry-org: os-climate
```
Should all of these be indented to the same level?